assets/js/components/ErrorHandler/index.js

```diff
@@ -6,13 +24,16 @@ import PropTypes from 'prop-types';
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
+import { Component, createRef } from '@wordpress/element';
+import { Dashicon } from '@wordpress/components';
```

Comment:
I think this may be the reason the bundle size went over the threshold – we should prefer using `@wordpress/icons` over Dashicons though as dashicons are somewhat "legacy" as of WP 5.5.

After refactoring with the icon library, the bundle size is no longer an issue 👍